### PR TITLE
Array Operations  

### PR DESCRIPTION
**Pull request by SANJAY-NT**
## Related Issue  

- Info about the related issue 

- [x] DevIncept Participant 
- [x] Contributor

Closes: #5 

### Describe the changes you've made

Created separate files to and implemented array operations like create an array, inserting an element to array , deleting an element and finally accessing the elements in the array. 
## Type of change

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made
  The programs were run on a pycharm IDE on localhost multiple time and checked for issues and error.
## Checklist:

- [X] My code follows the guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] I have commented my code, particularly whereever it was hard to understand.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.

### CODE REVIEW

**Comment by prathimacode-hub** [comment]
Add up your issue number in your PR template. @SANJAY-NT

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
What's this? What's the .idea folder, it's not needed. And the operations you had done should be in single project file. Not multiple files like this, as you had choosen Array operations. 

1 Issue = 1 PR
So Array Operations = 1 Array Operations project.
Also your README.md lacks details. Kindly work on it and be elaborative. 

Kindly do the changes accordingly. @SANJAY-NT

**Comment by SANJAY-NT** [comment]
Yes, in my issue i said I will add basic operations so i added only basic ones.

**Comment by prathimacode-hub** [comment]
> Yes, in my issue i said I will add basic operations so i added only basic ones.

Yes, but it should be only project file and other related files. Since it's 1 issue. Not multiple project files. @SANJAY-NT

**Comment by SANJAY-NT** [comment]
Ok I shall make one file and add all files under it , How do I edit in pull request. Any resources to refer.

File: Arrays/README.md [hunk 1]
```diff
@@ -1 +1,36 @@
+# Array
```
Comment (SANJAY-NT) [inline Arrays/README.md:2]: - An array is a collection of similar type items stored at contiguous memory locations.  This makes it easier to calculate the position of each element by simply adding an offset to a base value, i.e., the memory location of the first element of the array. The base value is index 0 and the difference between the two indexes is the offset. 
- In python array is implemented using list data structure.

**Review by SANJAY-NT — COMMENTED** [review]
I updated the readme and deleted all .idea and .git files.

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Changes to be done:
You are not getting my point. The project title should be as "Array Operations"
In this project folder, there should be "file_name.py", "requirements.txt", "README.md" and "Images" folder with output screenshots present in it.
And README.md should be elaborative, check with the template of the things you had missed including. 
There should be only 1 project file. That single .py file should compile all the operations of append, insert, pop and remove.
The project file should be as "array_operations.py"

You would have understood, if you had gone through the project structure in README file of this project. Hope you understood now of how PR should be and where you are going wrong. Kindly do the needful changes.

@SANJAY-NT

**Comment by prathimacode-hub** [comment]
> I am not able to move all to one file should I make changes to local repo and make a new pull request.

You don't have to create new PR, just do the changes in your forked repository,remove unnecesary things and push it again. And one more thing you should not move all the files into single file, instead you need to reprogram the project file which does all the operations in single .py file.

**Review by SANJAY-NT — COMMENTED** [review]
New changes implemented request review.

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Changes to be done:
The project file should be titled as "array_operations.py"
The requirements file should be titled as "requirements.txt". And requirements file means listing out the libraries included in this project along with the tools. Not prerequisites.
In README.md, add setup instructions, 3-4 test cases, compilation steps. And also the image link you added in 'user-images' content instead it should be the file path of the image present in this repo.

Kindly do these changes. @SANJAY-NT

**Review by prathimacode-hub — COMMENTED** [review]
Your README.md needs more details. You had missed few things. Kindly check it out. @SANJAY-NT

**Comment by SANJAY-NT** [comment]
I have still not completed the read me. I had kept auto commit on my IDE so it was updated.

**Comment by prathimacode-hub** [comment]
Ok, update me once you do it off. @SANJAY-NT

**Review by SANJAY-NT — COMMENTED** [review]
All Changes implemented and Updated.

**Review by prathimacode-hub — APPROVED** [review]
Congrats on your first PR. Yayy, you did it. This was a great start. Welcome to the repo officially. Your PR is approved. 🎉✨👏@SANJAY-NT

**Pull request merged**